lib/__tests__/InstantSearch-test.js

```diff
@@ -176,4 +176,56 @@ describe('InstantSearch lifecycle', () => {
       expect(algoliasearchHelper.args[0][2].values).toEqual([-2, -1, 0, 1, 3, 4, 2]);
     });
   });
+
+  context('when render happens', () => {
+    var renderCalled;
+    beforeEach(() => {
+      renderCalled = 0;
```

Comment:
And you can also check that render event was called AFTER render call :D `expect(renderCalled.calledBefore(renderEmitter));`